src/main/java/com/paneedah/weaponlib/WeaponSpawnEntity.java

```diff
@@ -183,4 +189,19 @@ public void shoot(final double x, final double y, final double z, final float ve
     public Weapon getWeapon() {
         return weapon;
     }
+
+    public static class ProjectileDamageSource extends EntityDamageSourceIndirect {
+        public ProjectileDamageSource(Entity projectile, @Nonnull Entity shooter) {
+            super("arrow", projectile, shooter);
```

Comment:
Once again - all of this is trying to use an arrow as the projectile damage source.
Please stop.

Reply:
The source of damage is not an arrow, it's just that the type of death message is an arrow